### PR TITLE
Issue 9 - Accessing time

### PR DESCRIPTION
**Pull request by bjorgan**
This should fix issue #9. Functions for converting to and from julian date and time_t are now available, and should replace the need for CurrentDayNum() by time(NULL).

### CODE REVIEW

**Comment by ryeng** [comment]
@bjorgan gmtime_r should be available if you #define _POSIX_C_SOURCE 1

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -22,6 +22,8 @@ list(APPEND predict_public_header
   ${LIBPREDICT_INCLUDE_DIR}/predict/orbit.h
   ${LIBPREDICT_INCLUDE_DIR}/predict/observer.h
   ${LIBPREDICT_INCLUDE_DIR}/predict/tle.h
+  ${LIBPREDICT_INCLUDE_DIR}/predict/juliandate.h
+  ${LIBPREDICT_INCLUDE_DIR}/predict/predict.h
```
Comment (ryeng) [inline src/CMakeLists.txt:22]: I'm not too happy about adding more header files to the interface. We could just put everything in predict.h and include that everywhere, or split into two: typedefs/structs and functions.

Comment (bjorgan) [inline src/CMakeLists.txt:22]: I agree. I mashed all public headers into one in commit ab1a0b92df554758c498adedc5f9d33491f4e448.

**Comment by ryeng** [comment]
LGTM.

**Pull request merged**